adafruit_si5351.py

```diff
@@ -192,9 +193,12 @@ def configure_fractional(self, multiplier, numerator, denominator):
             multiplier and numerator/denominator.  This is less accurate and
             susceptible to jitter but allows a larger range of PLL frequencies.
             """
-            assert 14 < multiplier < 91
-            assert 0 < denominator <= 0xFFFFF  # Prevent divide by zero.
-            assert 0 <= numerator < 0xFFFFF
+            if multiplier >= 91 or multiplier <= 14:
+                raise Exception("Multiplier must be in range 14 to 91.")
+            if denominator > 0xFFFFF or denominator <= 0:  # Prevent divide by zero.
+                raise Exception("Denominator must be in range 0 to 0xFFFFF.")
```

Comment:
For the message `range 0 to 0xFFFFF` implies it could be `0` or `0xFFFF`. How about `Denominator must be greater than 0 and less than 0xFFFFF`? Or `Denominator must be in range 1 to 0xFFFE`?

Reply:
Yes I was thinking about that, but was kind of in doubt because the code says one thing and the Exception says another, I explored other possibility to change the values in the if statement, but I said to myself, well if we do this, then we are not aligned with the Datasheet :).   But yes I will changed as suggested!! Thanks

Reply:
My comment could be wrong for the high end. Def can't be `0` on the low end. For the high end, just whatever the datasheet says, or whatever actually sets the limit.

Reply:
This was solved in dcf6fc7